src/debug/GdbCmds.cpp

```diff
@@ -46,7 +46,7 @@ const char* TARGET_INFO_ARM9 = "cputype:12;cpusubtype:7;triple:arm-none-eabi;ost
 	"<reg name=\"r11\" bitsize=\"32\" type=\"uint32\"/>" \
 	"<reg name=\"r12\" bitsize=\"32\" type=\"uint32\"/>" \
 	"<reg name=\"sp\" bitsize=\"32\" type=\"data_ptr\"/>" \
-	"<reg name=\"lr\" bitsize=\"32\" type=\"code_ptr\"/>" \
```

Comment:
lr definitely points to code (most of the time)?

Reply:
The situation is like this: when using IDA Pro for debugging, its view will follow the first register of type code_ptr. In the previous code, it would follow the lr register, causing issues with the view display during debugging.

Reply:
That does sound kind of stupid, though I guess you could move pc up?